### ISSUE TITLE
Annual check-in

### ISSUE THREAD

**Issue by Inquisitive117** [gibbed/Gibbed.MassEffectAndromeda#5]
Is this still on the "when I can revisit, I will" list?

**Comment by gibbed** [comment]
Yes, especially with the Legendary Edition and stuff releasing. I'm still very interested in revisiting.

**Comment by Inquisitive117** [comment]
Great to hear, thanks so much!

**Comment by JReming85** [comment]
Let me know if you we can do anything to help

**Comment by elchmartin** [comment]
just used this ... sometimes my saves load with the changes, sometimes changes are ignored ... 

would love to see thsi tool progressing!
and am willing to help if I can (testing etc..)
